views/includes/check-your-answers/beneficial-owner-gov.html

```diff
@@ -37,6 +37,16 @@
   {% endif %}
 {% endset %}
 
+{% if hiddenChangeLink == true %}
+  {% set changeLink = undefined %}
+{% else %}
+  {% set changeLink = CREATE_CHANGE_LINK(
+            changeLinkUrl + "/" + boc.id + OE_CONFIGS.START_DATE,
+            "Corporate beneficial owner " + boc.name + " - date it became a beneficial owner",
```

Comment:
strange - it's boc here and not bog as below

Reply:
yeah, silly mistake from me, will also revert the what I did to start date in the test controller, thanks!